client/rest-high-level/src/test/java/org/opensearch/client/RestHighLevelClientTests.java

```diff
@@ -885,7 +885,8 @@ public void testApiNamingConventions() throws Exception {
             "nodes.hot_threads",
             "nodes.usage",
             "nodes.reload_secure_settings",
-            "search_shards", };
+            "search_shards",
+            "remote_store.restore", };
```

Comment:
New API is added as high level rest client is not added for `_remotestore` APIs. I am not sure about guideline on writing high level rest client for new endpoints. If it is required, will add a tracking issue to add the same.

Reply:
I am guessing that will be needed since it's a new API. And also https://github.com/opensearch-project/opensearch-java and https://github.com/opensearch-project/opensearch-api-specification. I'd appreciate a write up on "adding new RESTful APIs" somewhere in the developer guide, and all the subsequent changes that are required today for the next person.

Reply:
Tracking issues: https://github.com/opensearch-project/OpenSearch/issues/3602 and https://github.com/opensearch-project/opensearch-java/issues/168